transhumandesign/kag-base
# Improve accolade_data.cfg format

## Status
**READY**

## Description
Changes Accolades.as and accolade_data.cfg to accommodate the new format described in #1112.

accolade_data.cfg is up to date with the latest changes as of 13/11/2021

The script that formatted the file: https://github.com/laws65/accolade-data-converter

## Steps to test:
- Join a local server
- Check if your accolades appear correctly
- Exit the game
- Modify accolade_data.cfg to change what accolades you have
- Repeat
